src/abstractdatatable/abstractdatatable.jl

```diff
@@ -801,6 +750,165 @@ function Base.hash(dt::AbstractDataTable)
     return @compat UInt(h)
 end
 
+"""
+    denullify!(dt::AbstractDataTable)
+
+Convert columns with a `Nullable` element type without any null values
+to a non-`Nullable` equivalent array type. The table `dt` is modified in place.
```

Comment:
I'm not sure, we would need concrete use cases to decide. In both cases people can easily make a copy manually.